addon/webextension/background/main.js

```diff
@@ -295,5 +179,17 @@ this.main = (function() {
     return startSelectionWithOnboarding(sender.tab);
   });
 
+  // Open a port to let the embedding legacy addon tell us when a button is pressed
+  // TODO: do we need to specify the extensionID in the connect call?
```

Comment:
stale todo - the console.log is fired when the button is clicked repeatedly, so this code does work